picobot/painter.py

```diff
@@ -44,17 +46,17 @@
 }
 
 
-def draw_balloon(img_draw: ImageDraw.Draw, points: Box, fill=None, width=0):
+def draw_balloon(img_draw: ImageDraw.ImageDraw, points: Box, fill=None, width=0):
```

Comment:
Any specific reason to not explicit the return type in this file's functions?

Reply:
No, maybe mypy should be set to stricter configs tho.